cms/djangoapps/contentstore/views/item.py

```diff
@@ -552,7 +552,10 @@ def _duplicate_item(parent_usage_key, duplicate_source_usage_key, user, display_
         category = dest_usage_key.block_type
 
         # Update the display name to indicate this is a duplicate (unless display name provided).
-        duplicate_metadata = own_metadata(source_item)
+        duplicate_metadata = {}
```

Comment:
Thank you for the detailed explanation-- that helped. Please just include a brief comment in the code stating that we cannot use own_metadata because of the intermediate representation issue (which would break LibraryList, for example).